radio/src/opentx.cpp

```diff
@@ -469,7 +469,7 @@ void modelDefault(uint8_t id)
 #endif
 
 #if defined(PCBXLITE)
-  g_model.trainerMode = TRAINER_MODE_MASTER_BLUETOOTH;
+  g_model.trainerMode = TRAINER_MODE_MASTER_CPPM_EXTERNAL_MODULE;
```

Comment:
Maybe better to check if BT is present, leave BT as default if yes, set this if not present ?